unittests/test_policies.py

```diff
@@ -34,15 +34,20 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree(self.resourcesdir, ignore_errors=True)
 
+    def _num_failures_stage(self, stage):
+        stats = self.runner.stats
+        return len([t for t in stats.tasks_failed() if t.failed_stage ==
+                                                         stage])
```

Comment:
Fix the line split here. Move the whole `if` statement in the next line.

Reply:
Done.